app/models/price_table.rb

```diff
@@ -0,0 +1,5 @@
+class PriceTable < ApplicationRecord
+    validates_presence_of :trainees, :price_point
+    has_many :sessions
+    enum price: { low: 0, medium: 1, high: 2 }
```

Comment:
This should be the same name as in the db table: `price_point`

Reply:
Done